tools/setup_anaconda.sh

```diff
@@ -57,6 +58,8 @@ fi
 conda install -y pip setuptools
 
 cat << EOF > activate_python.sh
+#!/bin/bash
+# THIS FILE IS GENERATED BY tools/setup_anaconda.sh
```

Comment:
Would you also write these lines in tools/setup_venv.sh ?

Reply:
Done